src/sentry/models/projectownership.py

```diff
@@ -276,37 +296,25 @@ def handle_auto_assignment(cls, project_id, event=None, group=None):
             except (User.DoesNotExist, Team.DoesNotExist):
                 return
 
-            details = (
-                {"integration": ActivityIntegration.SUSPECT_COMMITTER.value}
-                if issue_owner.type == GroupOwnerType.SUSPECT_COMMIT.value
-                else (
-                    {
-                        "integration": ActivityIntegration.PROJECT_OWNERSHIP.value,
-                        "rule": (issue_owner.context or {}).get("rule", ""),
-                    }
-                    if issue_owner.type == GroupOwnerType.OWNERSHIP_RULE.value
-                    else {
-                        "integration": ActivityIntegration.CODEOWNERS.value,
-                        "rule": (issue_owner.context or {}).get("rule", ""),
-                    }
-                )
-            )
+            activity_details = {}
+            if issue_owner.type == GroupOwnerType.SUSPECT_COMMIT.value:
+                activity_details["integration"] = ActivityIntegration.SUSPECT_COMMITTER.value
+            elif issue_owner.type == GroupOwnerType.OWNERSHIP_RULE.value:
+                activity_details["integration"] = ActivityIntegration.PROJECT_OWNERSHIP.value
+                activity_details["rule"] = (issue_owner.context or {}).get("rule", "")
+            else:
+                activity_details["integration"] = ActivityIntegration.CODEOWNERS.value
+                activity_details["rule"] = (issue_owner.context or {}).get("rule", "")
```

Comment:
I would have put these directly on `logging_extra` but turns out they are also saved to the database on the `Activity` object's `data` field 🔍